.github/workflows/cypress.yaml

```diff
@@ -1,13 +1,33 @@
 # Triggers after the layered build has finished, taking the artifact and running cypress on it
+#
+# Also called by a workflow in matrix-js-sdk.
+#
 name: Cypress End to End Tests
 on:
     workflow_run:
         workflows: ["Element Web - Build"]
         types:
             - completed
+
+    # support calls from other workflows
+    workflow_call:
+        inputs:
+            react-sdk-repository:
+                type: string
+                required: true
+                description: "The name of the github repository to check out and build."
```

Comment:
Surely this could be driven by just choosing that repository to workflow_call an action upon?

Reply:
Sorry, I don't understand?

Reply:
To be clear: I don't believe there is any way to figure out which repo this workflow file has been sourced from, from within the workflow.

Reply:
Sure, and each repo/fork could just hardcode their own repo - https://github.com/matrix-org/matrix-react-sdk/blob/develop/.github/workflows/tests.yml#LL36C21-L36C21

Reply:
a third approach is to split out into a composite action which the js-sdk and other callers can call, and it'll have access to its own repository via `github.action_repository` where no duplication or hard-coding is needed